components/ui-shell/index.scss

```diff
@@ -62,30 +62,53 @@ $prefix: "bx";
   }
 }
 
-:host(#{$q}-header-name) {
+#{$prefix}-header-name {
   &::part(link) {
     width: 8.75rem;
     min-height: var(--header-height);
     fill: var(--cool-gray-80);
   }
 }
 
-:host(#{$q}-header-nav) {
-  /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
-  @extend :host(#{$prefix}-header-nav);
-
+#{$prefix}-header-nav {
+  &::before,
   &::part(divider) {
     display: none;
   }
 }
 
-:host(#{$q}-header-nav-item.#{$q}-user-accout-icon) {
-  a {
+#{$prefix}-header-nav-item.#{$q}-user-accout-icon {
+  &::part(link) {
```

Comment:
Noticing the account icon turn white on focus/hover. I think this should cover the states
<img width="455" alt="Screen Shot 2022-09-09 at 7 24 50 AM" src="https://user-images.githubusercontent.com/8097306/189375862-57345eb5-2f59-4db8-8a63-f34269467d62.png">



```suggestion
  &::part(link):focus,
  &::part(link):hover,
  &::part(link) {
```